.github/workflows/official-pr.yml

```diff
@@ -11,7 +11,6 @@ on:
       - "**/architectures"
       - "**/docker-entrypoint.sh"
       - "stackbrew.js"
-      - "config"
```

Comment:
should we delete the file?

Reply:
It's still used by the functions.sh/update.sh right now